pkg/imgpkg/bundle/contents_test.go

```diff
@@ -44,7 +44,7 @@ images:
 			t.Fatalf("failed to read tag: %s", err)
 		}
 
-		_, err = subject.Push(imgTag, fakeRegistry, util.NewNoopLevelLogger())
+		_, err = subject.Push(imgTag, make(map[string]string), fakeRegistry, util.NewNoopLevelLogger())
```

Comment:
```suggestion
		_, err = subject.Push(imgTag, map[string]string{}, fakeRegistry, util.NewNoopLevelLogger())
```
for consistency with next change

Reply:
Ah, yeah, I had that originally and was testing some things and forgot to change it back.